tests/integration/gather_plugin/dotnet/MyStack.cs

```diff
@@ -7,10 +7,10 @@ public MyStack()
         // Create an AWS resource (S3 Bucket)
         var r = new Random(
             "default", 10, new ComponentResourceOptions{
-                PluginDownloadURL = "get.com",
+                PluginDownloadURL = "get.example.test",
```

Comment:
This applies to other test-content changes, but what's the reason for this change?

Reply:
I really don't like it when uncontrolled third party URLs/domain names are in tests. If everything is correctly mocked it's fine, but if not suddenly dev machines & CI are making requests to who-knows-what. 

It's even scarier here because these would potentially cause us to download and install a tar.gz file in CI! I checked and that doesn't happen here, but whew, kind of scary.

The IETF reserved `.test` as a top-level domain that will **never** be registered or respond to DNS. (Same with `.example`, `.invalid`, a few others.)